lib/common.js

```diff
@@ -1,51 +1,51 @@
 'use strict';
 
-module.exports.aliasResources = function (resourceObject) {
-  // We support the full names and all the abbbreviated aliases:
-  //   http://kubernetes.io/docs/user-guide/kubectl-overview/
-  // and anything else we think is useful.
-  const resourceAliases = {
-    clusterroles: [],
-    clusterrolebindings: [],
-    componentstatuses: ['cs'],
-    configmaps: ['cm'],
-    cronjobs: [],
-    customresourcedefinitions: ['crd'],
-    daemonsets: ['ds'],
-    deployments: ['deploy'],
-    events: ['ev'],
-    endpoints: ['ep'],
-    horizontalpodautoscalers: ['hpa'],
-    ingresses: ['ing'],
-    jobs: [],
-    limitranges: ['limits'],
-    namespaces: ['ns'],
-    nodes: ['no'],
-    persistentvolumes: ['pv'],
-    persistentvolumeclaims: ['pvc'],
-    // Deprecated name of statefulsets in kubernetes 1.4
-    petsets: [],
-    pods: ['po'],
-    replicationcontrollers: ['rc'],
-    replicasets: ['rs'],
-    resourcequotas: ['quota'],
-    roles: [],
-    rolebindings: [],
-    // Deprecated name of cronjobs in kubernetes 1.4
-    scheduledjobs: [],
-    secrets: [],
-    serviceaccounts: [],
-    services: ['svc'],
-    statefulsets: [],
-    // Deprecated name of customresourcedefinition in kubernetes 1.7
-    thirdpartyresources: []
-  };
+// We support the full names and all the abbbreviated aliases:
+//   http://kubernetes.io/docs/user-guide/kubectl-overview/
+// and anything else we think is useful.
+const resourceAliases = {
+  clusterroles: [],
+  clusterrolebindings: [],
+  componentstatuses: ['cs'],
+  configmaps: ['cm'],
+  cronjobs: [],
+  customresourcedefinitions: ['crd'],
+  daemonsets: ['ds'],
+  deployments: ['deploy'],
+  events: ['ev'],
+  endpoints: ['ep'],
+  horizontalpodautoscalers: ['hpa'],
+  ingresses: ['ing'],
+  jobs: [],
+  limitranges: ['limits'],
+  namespaces: ['ns'],
+  nodes: ['no'],
+  persistentvolumes: ['pv'],
+  persistentvolumeclaims: ['pvc'],
+  // Deprecated name of statefulsets in kubernetes 1.4
+  petsets: [],
+  pods: ['po'],
+  replicationcontrollers: ['rc'],
+  replicasets: ['rs'],
+  resourcequotas: ['quota'],
+  roles: [],
+  rolebindings: [],
+  // Deprecated name of cronjobs in kubernetes 1.4
+  scheduledjobs: [],
+  secrets: [],
+  serviceaccounts: [],
+  services: ['svc'],
+  statefulsets: [],
+  // Deprecated name of customresourcedefinition in kubernetes 1.7
+  thirdpartyresources: []
+};
 
-  const esPlurals = {
-    componentstatuses: true,
-    ingresses: true
-  };
+const esPlurals = {
+  componentstatuses: true,
+  ingresses: true
+};
 
+module.exports.aliasResources = function (resourceObject) {
```

Comment:
It's a mix, and not a very well defined convention: arrow functions for callbacks, function expressions for everything else.  It could make sense to revisit that "convention" in another PR / issue.